### PR TITLE
added missing # and fixed readme

### PR DESCRIPTION
**Pull request by QBY-ChristianHartmann**
# Description

As discussed in #(issue) we need to support xxx. Therefore we changed yyy.

# Change overview (tick true):

- [ ] This introduces backward incompatible changes
- [ ] This adds a new backward compatible Feature
- [x] This fixes a Bug

# Version information: 

- Previous Version: `major.minor.patch`

- Next Version based on [Semantic Versioning](https://semver.org/#summary) (see above): `major.minor.patch`

# How Has This Been Tested?

- [x] Apply of all examples was successfull
- [ ] Test B

# Checklist:

- [ ] I have run tests and documented them above
- [x] I have performed a self-review of my own code
- [ ] I have updated the documentation
- [ ] I have updated the CHANGELOG

### CODE REVIEW

**Review by schnaddy — DISMISSED** [review]
Loogs good! Great job

File: .terraform-docs.yml [hunk 1]
```diff
@@ -43,32 +43,17 @@ content: |-
...
-
-  ## Examples
-
-  ### Basic
+   - Personal Access Token for the DevOps Organization to create service connections and repositories 
+   - Project Admin on DevOps Project
+   - Admin User to create the service principal in the Customer Tenant. If you want to move the subscription into a new management group you need an admin user directly in the tenant. AOBO will not work.
 
   This is a basic terraform.tfvars template to use
   ```hcl
```
Comment (PLN-Qbeyond) [inline .terraform-docs.yml:56]:
```suggestion

```

File: .terraform-docs.yml [hunk 2]
```diff
@@ -43,32 +43,17 @@ content: |-
...
-  {{ include "examples/basic/terraform.tfvars.tmpl" }}
   ```
```
Comment (PLN-Qbeyond) [inline .terraform-docs.yml:57]:
```suggestion

```

**Review by schnaddy — COMMENTED** [review]
diese zeilen werden nicht gebraucht

**Review by ghost — APPROVED** [review]

**Pull request merged**